client/js/verification-portal/components/header-modal.js

```diff
@@ -44,40 +47,34 @@ class HeaderModal extends Component {
 			this.state.closed && classes.closed,
 		)
 
-		if (
-			(submitStatus === 'Success' && this.state.prevSubmit === null) ||
-			this.state.prevSubmit === 'Success'
-		) {
-			return (
-				<div className={modalClasses}>
-					<div>
-						<p className={classes.title}>Information Submitted Successfully</p>
-						<p className={classes.caption}>
-							{
-								"Thanks for helping us bring better healthcare to women's fingertips!"
-							}
-						</p>
-					</div>
-					<div className={classes.button} onClick={this.handleClick}>
-						&times;
-					</div>
+		let title
+		let caption
+		if (submitStatus === 'Success') {
+			title = 'Information Submitted Successfully'
+			caption =
+				"Thanks for helping us bring better healthcare to women's fingertips!"
+		} else if (submitStatus === 'Failed') {
+			title = 'Whoops!'
+			caption = 'An error occurred and the HAH staff has been notified.'
+		}
+
+		return (
+			<div className={modalClasses}>
```

Comment:
This code is basically the same. The only thing that is different is the wording for the title and caption. We can simplify most of this code by defining the caption and title (above) in variables.